.github/workflows/build_bench.yml

```diff
@@ -22,13 +22,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: all
+
       - name: Setup Buildx
         uses: docker/setup-buildx-action@v1
 
       - name: Build and test
         uses: docker/bake-action@v1.7.0
         with:
           targets: bench-test
+          set: "*.platform=${{ github.event_name != 'pull_request' && 'linux/amd64,linux/arm64' || 'linux/amd64' }}"
```

Comment:
Set platforms on Push step and drop the condition.

Reply:
done